frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/common.ts

```diff
@@ -19,14 +19,8 @@ import { getTemplateOperatingSystems } from '../../../../selectors/vm-template/a
 import { VMTemplateWrapper } from '../../../wrapper/vm/vm-template-wrapper';
 import { operatingSystemsNative } from '../../../../components/create-vm-wizard/native/consts';
 import { concatImmutableLists, immutableListToShallowJS } from '../../../../utils/immutable';
-import { CreateVMParams } from './types';
 
-export const getOS = ({
-  vmSettings,
```

Comment:
we should just pass the osID here and not the whole vmSettings I guess

Reply:
Done!